Added header to the Jobmarket

Actionmaker1 opened the pull request:
Added general header, looks good on mobile view. Couldnt interpolate career or step into the heading, not sure why. I can work on that after lunch or when I'm done with the rest

Actionmaker1 committed:
added header to the Jobmarket

dwiehoff commented on app/assets/stylesheets/pages/_index.scss:
```diff
@@ -4,3 +4,4 @@
 @import "compare";
 @import "questions";
 @import "roadmap";
+@import "job"
```
missing semicolon here

dwiehoff committed:
Update jobs.html.erb

Rephrased to be in line with functionality (latter jobs wouldn't match experience)

dwiehoff committed:
Update _index.scss

Added semicolon

dwiehoff reviewed (approved):
Made adjustments, ready to merge
